qa/rolling-upgrade/src/test/resources/rest-api-spec/test/upgraded_cluster/10_basic.yml

```diff
@@ -7,6 +7,7 @@
         # wait for long enough that we give delayed unassigned shards to stop being delayed
         timeout: 70s
         level: shards
+        index: test_index, index_with_replicas, multi_type_index
```

Comment:
why do we need to wait here? Why not directly in the corresponding tests? This makes for example the RecoveryIT test dependent on a specific yml file, which is ugly IMO.

Reply:
I misread, ignore this comment.